lib/install/validate/validator.go

```diff
@@ -222,6 +240,10 @@ func (v *Validator) target(ctx context.Context, input *data.Data, conf *config.V
 			v.NoteIssue(fmt.Errorf("Error processing target after transformation to SOAP endpoint: %q: %s", v.Session.Service, err))
 			return
 		}
+
+		// ESXi requires user/password to be encoded in the Target URL
```

Comment:
The connection parameter is set here: https://github.com/vmware/vic/blob/master/lib/install/management/appliance.go#L435
Please reference what port-layer did: https://github.com/vmware/vic/blob/master/lib/install/management/appliance.go#L494
The URL does not lose password information if it's there before serialize.